cmd/util/cmd/read-hotstuff/cmd/get_liveness.go

```diff
@@ -19,28 +21,35 @@ func init() {
 }
 
 func runGetLivenessData(*cobra.Command, []string) {
-	db := common.InitStorage(flagDatadir)
-	defer db.Close()
+	err := common.WithStorage(common.DBDirs{
+		Datadir:   flagDatadir,
+		Pebbledir: flagPebbleDir,
+	}, func(db storage.DB) error {
+
+		chainID := flow.ChainID(flagChain)
+		reader, err := persister.NewReader(db, chainID)
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not create reader from db")
+		}
+
+		log.Info().
+			Str("chain", flagChain).
+			Str("datadir", flagDatadir).
+			Str("pebbledir", flagPebbleDir).
+			Msg("getting hotstuff liveness data")
+
+		livenessData, err := reader.GetLivenessData()
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not get hotstuff liveness data")
+		}
+
+		log.Info().Msgf("successfully get hotstuff liveness data")
+		common.PrettyPrint(livenessData)
+		return nil
+	})
 
-	storages := common.InitStorages(db)
-	state, err := common.InitProtocolState(db, storages)
```

Comment:
We initialize the protocol state just to get the chain ID, I changed to get the chain ID from the flag, so that we can decouple from the protocol state.